stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamDimens.kt

```diff
@@ -162,7 +163,8 @@ public data class StreamDimens(
             quotedMessageAttachmentStartPadding = 8.dp,
             quotedMessageAttachmentEndPadding = 0.dp,
             groupAvatarInitialsXOffset = 1.5.dp,
-            groupAvatarInitialsYOffset = 2.5.dp
+            groupAvatarInitialsYOffset = 2.5.dp,
+            attachmentsContentImageMaxHeight = 600.dp,
```

Comment:
`attachmentsContentImageHeight` would be deprecated in this scenario. 

The idea would no longer be that users set a fixed height, but rather a maximum height that the images could expand to since now they adapt themselves according to the aspect ratio.

Reply:
I'm not 100% sure we receive height/width for legacy images, i.e. ones uploaded before the switch - could you check that.

If that is the case, then having a max height still could be useful.

Reply:
We don't, hence why [here](https://github.com/GetStream/stream-chat-android/pull/4013/files/0eae187d4fbe12b96bb7abc547034f084b3db96a#diff-26ca363b87ad5e9cb52b46ce1d0ae50ab0910042d042e3ca7cca6d051c74c21eR110) Ithe ratio is nullable and the fallback option is to square the photo using a ratio of `1:1`.

Essentially, this way our images follow the pattern of our group images so we're safe, even if slightly less aesthetically pleasing.